Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule.py

```diff
@@ -28,6 +29,8 @@ class Scopes:
 REGEX_SEARCH_URL = r'(?P<url>https?://[^\s]+)'
 SESSION_STATE = 'session_state'
 
+TOKEN_RETRIEVAL_URI = 'https://login.microsoftonline.com/{tenant_id}/oauth2/v2.0/token'
```

Comment:
Not necessary. If you choose to keep it, please use it in line 39 instead of the hard-coded value there.